components/CreateOrganization.js

```diff
@@ -118,21 +127,28 @@ class CreateOrganization extends React.Component {
                   collective={this.state.collective}
                   onSubmit={this.createCollective}
                   onChange={this.resetError}
+                  error={result.error}
+                  LoggedInUser={LoggedInUser}
```

Comment:
Instead of passing `LoggedInUser` down as a prop to the form component, call `withUser` in the form component itself (search the codebase to see examples)